poromechanics/use_cases/consolidation_interface_3D/source/PoroMaterials.json

```diff
@@ -0,0 +1,63 @@
+{
+    "properties": [{
+        "model_part_name": "PorousModelPart.Body_Part-auto-1",
+        "properties_id": 3,
+        "Material": {
+            "constitutive_law": {
+                "name": "LinearElasticSolid3DLaw"
+            },
+            "Variables": {
+                "YOUNG_MODULUS": 1.0e6,
+                "POISSON_RATIO": 0.2,
+                "DENSITY_SOLID": 0.0,
+                "DENSITY_WATER": 1.0e3,
+                "POROSITY": 0.3,
+                "BULK_MODULUS_SOLID": 1.0e12,
+                "BULK_MODULUS_FLUID": 2.0e9,
+                "PERMEABILITY_XX": 4.5e-10,
```

Comment:
Orthopermeability, nice. Question not related to this pR, is this registered one by one or using the macro for tensors?

Reply:
I think it’s registered one by one ^_^’
I know it can be done as a matrix, but if I remember correctly, I did it this way to make it more flexible...